pkgs/applications/version-management/sourcehut/builds.nix

```diff
@@ -11,26 +11,54 @@
 , python
 }:
 let
-  version = "0.66.7";
+  version = "0.73.3";
 
-  buildWorker = src: buildGoModule {
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = "builds.sr.ht";
+    rev = version;
+    sha256 = "sha256-/0KYQgl/pfZomD1EEF9iI9HnNR99a8OB3/n5IfgClQk=";
+  };
+
+  worker = buildGoModule {
     inherit src version;
-    pname = "builds-sr-ht-worker";
+    sourceRoot = "source/worker";
+    pname = "buildsrht-worker";
+
+    vendorSha256 = "sha256-7zlt5305P3KzGrs4wUyAU1+FpoBMyl+yjkLSzqUybxg=";
+
+    # What follows is only to update go-redis,
+    # and thus also using a patched srht-keys.
+    # go.{mod,sum} could be patched directly but that would be less resilient
+    # to changes from upstream, and thus harder to maintain the patching
+    # while it hasn't been merged upstream.
+
+    overrideModAttrs = old: {
+      preBuild = ''
+        go get github.com/go-redis/redis/v8
+        go get github.com/go-redis/redis@none
+        go mod tidy
+      '';
+      # Pass updated go.{mod,sum} from go-modules to worker's vendor/go.{mod,sum}
+      postInstall = ''
+        cp --reflink=auto go.* $out/
+      '';
+    };
 
-    vendorSha256 = "sha256-giOaldV46aBqXyFH/cQVsbUr6Rb4VMhbBO86o48tRZY=";
+    patches = [
+      # Update go-redis to support Unix sockets
+      patches/redis-socket/build/v3-0001-worker-update-go-redis-to-support-Unix-sockets.patch
```

Comment:
```suggestion
      patches/v3-0001-worker-update-go-redis-to-support-Unix-sockets.patch
```

Reply:
I've committed this, believing my path was wrong, but it was not, I'll revert.